### PR TITLE
Publish sources #177

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -51,6 +51,7 @@ node('rhel7'){
         dir ('vscode-quarkus') {
             def packageJson = readJSON file: 'package.json'
             sh "vsce package -o ../vscode-quarkus-${packageJson.version}-${env.BUILD_NUMBER}.vsix"
+            sh "npm pack && mv vscode-quarkus-${packageJson.version}.tgz vscode-quarkus-${packageJson.version}-${env.BUILD_NUMBER}.tgz"
```
Comment (apupier) [inline Jenkinsfile]: wondering why there is a ../ in previous line. Is it changing something here?

Comment (fbricon) [reply Jenkinsfile]: this step is running inside the vscode-quarkus directory. It's putting the vsix file in the parent dir, so you'll need to do the same

Comment (apupier) [reply Jenkinsfile]: Thanks for the explanation. I modified, to also move the generated tgz source files to the parent dir.

**Comment by fbricon** [comment]
@apupier thanks for the PR. Before merging, I ran a jenkins build (\#221)  with those changes, the sources have been pushed to https://download.jboss.org/jbosstools/vscode/snapshots/vscode-quarkus/?C=M;O=D

**Pull request merged**